packages/votingV2/src/mappings/votingV2.ts

```diff
@@ -38,10 +44,19 @@ export function handlePriceRequestAdded(event: PriceRequestAdded): void {
   let requestRound = getOrCreatePriceRequestRound(requestId.concat("-").concat(event.params.roundId.toString()));
 
   request.identifier = event.params.identifier.toString();
+  request.requestIndex = event.params.requestIndex;
   request.latestRound = requestRound.id;
   request.time = event.params.time;
-
+  request.ancillaryData = event.params.ancillaryData.toHex();
   request.isGovernance = event.params.isGovernance;
+
+  // If governance request we manually create an identifier as it doesn't exist on chain
+  if (event.params.isGovernance) {
```

Comment:
If it's a governance price request I decided to manually create an Identifier to be able to query it in the subgraph. Does it make sense to you @chrismaree ?